pfb-analysis/import/import_neighborhood.sh

```diff
@@ -45,9 +44,10 @@ NB_POSTGRESQL_PASSWORD - Default: gis
 function import_and_transform_shapefile() {
     IMPORT_FILE="${1}"
     IMPORT_TABLENAME="${2}"
+    IMPORT_SRID="${3:-4326}"
```

Comment:
They're in PR #119.  I was thinking it would be better to base changes off that version than have to merge later, but actually the changes I made don't overlap, so I'll remove them from this branch.